stores/sql/sqlite/main.go

```diff
@@ -711,6 +715,36 @@ func (tx *MainDatabaseTx) UpdateHostBlocklistEntries(ctx context.Context, add, r
 	return nil
 }
 
+func (tx *MainDatabaseTx) UpdateHostCheck(ctx context.Context, autopilot string, hk types.PublicKey, hc api.HostCheck) error {
+	_, err := tx.Exec(ctx, `
+	    INSERT INTO host_checks (created_at, db_autopilot_id, db_host_id, usability_blocked, usability_offline, usability_low_score,
+	        usability_redundant_ip, usability_gouging, usability_not_accepting_contracts, usability_not_announced, usability_not_completing_scan,
+	        score_age, score_collateral, score_interactions, score_storage_remaining, score_uptime, score_version, score_prices,
+	        gouging_contract_err, gouging_download_err, gouging_gouging_err, gouging_prune_err, gouging_upload_err)
+	    VALUES (?,
+			(SELECT id FROM autopilots WHERE identifier = ?),
```

Comment:
Bit of a nit I guess but is the error that's being returned when an autopilot can not be found clear enough to indicate what's happening? Some for the host obviously. Nested queries are fine by me but it's kind of annoying if the resulting error is vague.